boxsdk/object/event.py

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+from boxsdk.object.base_object import BaseObject
+
+
+class Event(BaseObject):
+    """Represents a single Box event"""
+
+    _item_type = 'event'
+
+    @property
+    def get_url(self, *args):
+        """
+        Base class override.
+        Disallow this method for this subclass, should not access Event by event_id.
+        """
+        raise AttributeError("'Event' class has no method 'get_url'")
```

Comment:
Yeah, I had discussed this with Jeremy because I wasn't sure what approach was best.  Most methods in `BaseObject` use get_url(), so I think the only behaviours of `BaseObject` that we actually want to mimic are __init__, __getitem__, __repr__, object_id, and __eq__.  
Is there an option besides reproducing those behaviors separately in `Event`?